### ISSUE TITLE
Remove unnecessary label on the patient experience header

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3976]
## Area or Component
Patient Experience

## Quick Summary
A div in the patient experience header is labelled as home, but a) it doesn't actually lead to the home and b) divs do not have a role and aren't recommended to have an aria-label.

## Accessibility Violation
Ensures ARIA attributes are allowed for an element's role: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome

## Action Requested
Remove the aria-label from the div. We may also want to remove the title.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/PatientHeader.tsx#L33-L33

## Original Deque Issue
aria-allowed-attr | Ensures ARIA attributes are allowed for an element's role | aria-label is used on a div element (home) that isn't actually clickable/interactable (it's the div for the logo/"Covid-19 testing portal" at the top of the page) | Elements must only use allowed ARIA attributes | https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome | serious | No | No |   | ["div[title=\"Home\"]"] | Fix all of the following:  aria-label attribute is not well supported on a div with no valid role attribute. | <div class="display-flex flex-align-center" title="Home" aria-label="Home"> | cat.aria,wcag2a,wcag412 | rnv7@cdc.gov | Patient Self-Registration - Form | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T21:45:34.042Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
